packages/core/src/DatePicker/DatePicker.stories.tsx

```diff
@@ -303,13 +300,13 @@ export const Controlled: StoryObj<HvDatePickerProps> = {
 
 export const WithSelectionList: StoryObj<HvDatePickerProps> = {
   parameters: {
-    eyes: {
-      runBefore() {
-        fireEvent.click(screen.getByRole("combobox"));
-
-        return waitFor(() => screen.getByRole("button", { name: "Apply" }));
```

Comment:
Isn't a similar logic to this wait needed to make sure the image is taken when the date picker is open?
This comment is applicable in other places.

Reply:
Looking at the image taken by Chromatic it does not seem needed:
![image](https://github.com/lumada-design/hv-uikit-react/assets/14977550/16875957-3343-4e7f-9dfe-fe32f5f2a976)